fv_be/app/admin/base_admin.py

```diff
@@ -70,16 +71,30 @@ def item_id(self, instance):
     item_id.short_description = _("Id")
 
     def admin_link(self, instance):
-        url = reverse(
-            f"admin:{instance._meta.app_label}_{instance._meta.model_name}_change",
-            args=(instance.id,),
-        )
-        # todo: i18n for 'Edit' not working here for some reason
-        return format_html('<a href="{}">{}: {}</a>', url, "Edit", str(instance))
+        # todo: Add reverse links for all models
+        try:
+            url = reverse(
+                f"admin:{instance._meta.app_label}_{instance._meta.model_name}_change",
+                args=(instance.id,),
+            )
+            # todo: i18n for 'Edit' not working here for some reason
+            return format_html('<a href="{}">{}: {}</a>', url, "Edit", str(instance))
+        except NoReverseMatch as e:
+            return None
 
     def save_model(self, request, obj, form, change):
         if not change:
             obj.created_by = request.user
 
         obj.last_modified_by = request.user
         super().save_model(request, obj, form, change)
+
+
+# todo: better name for following class
+class AdminHideUtility(BaseAdmin):
```

Comment:
Let's call this HiddenBaseAdmin or something similar, to make it clear that it is a base admin class.